code/game/machinery/doors/airlock.dm

```diff
@@ -456,8 +457,8 @@
 		return TRUE
 	return ((!secondsMainPowerLost || !secondsBackupPowerLost) && !(stat & NOPOWER))
 
-/obj/machinery/door/airlock/requiresID()
-	return !(wires.is_cut(WIRE_IDSCAN) || aiDisabledIdScanner)
+/obj/machinery/door/airlock/id_scan_hacked()
+	return wires.is_cut(WIRE_IDSCAN) || aiDisabledIdScanner
```

Comment:
This could be a define

Reply:
It could, but should it? This isn't a performance-critical part of the code, and defines that aren't trivial expressions add size overhead. But this is an area where you have more experience and I'm less aware of the issues/limitations of DM, plus the general coding standards - if you still think it's the right call, I'll do it.